src/common/signalk/KMessageNMEAVisitor.cpp

```diff
@@ -35,10 +35,10 @@ void KMessageNMEAVisitor::visit(const NMEA2000Message &n2km) {
 
   const tN2kMsg& msg = n2km.getN2kMsg();
 
+  // TODO: integrate SDCardConfig to check if SeaSmart is enabled
```

Comment:
You could just pass a boolean to this visitor constructor. When we create it we know if we need to generate PCDIN or not.

Reply:
I did not give up the idea, that KBox could also write Actisense Binary to Serial output or WiFi....
It´s something for future works, if we get bored and have nothing else to do :-)

But for now a boolean should make it perfectly!